.github/actions/is-release-week-v1/tsconfig.json

```diff
@@ -0,0 +1,7 @@
+{
+  "extends": "../../../tsconfig.json",
+  "compilerOptions": {
+    "rootDir": "."
+  },
```

Comment:
```suggestion

```

This can be removed the [root confg](https://github.com/dequelabs/axe-api-team-public/blob/main/tsconfig.json#L8) points to `src`. This is currently causing your `dist` to look like: 

```
dist
	index.d.ts
	utils.d.ts
	index.js
	licenses.txt
src
	index.d.ts
	utils.d.ts
```

`ncc` builds everything into 1 JS file so we should only see this in `dist`: 
```
dist
	index.d.ts
	utils.d.ts
	index.js
	licenses.txt
```